app/src/main/java/com/example/weatherapp/Api.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.weatherapp
+
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface Api {
+    @GET( "weather")
+    fun getCurrentConditions(
+        @Query( "zip") zip: String,
+        @Query( "units") units: String = "Imperial",
+        @Query( "appId") appId: String = "5e7d9b53cd79cc03a42cb84a604eb799",
+    ) : Call<CurrentConditions>
+
+    @GET( "daily")
```

Comment:
should be `@GET("forecast/daily")` so you can have the same base urls in your retrofit instance.